app/routes/privacy-policy/route.module.scss

```diff
@@ -0,0 +1,3 @@
+.heading2 {
+    font-weight: 400;
+}
```

Comment:
Why here rather than than in `.textPage h2`?

Reply:
No particular reason, I will change